src/Version.ts

```diff
@@ -12,4 +12,4 @@
  * @module Version
  */
 
-export const PACKAGE_VERSION = "2.0.0";
+export const PACKAGE_VERSION = "2.1.0";
```

Comment:
I want to verify that we are working with the same code as the 2.1.0-preview.1 release. I expected to see this change from `2.1.0-preview.1` to `2.1.0`. What approach did you use to make this change?

Reply:
I branched of the tag [2.1.0-preview.1](https://github.com/microsoftgraph/msgraph-sdk-javascript/tree/2.1.0-Preview.1) and then changed the version in package.json which is used to set version and created the PR against master.

The preview releases are made from dev. Since dev contains changes aside from  2.1.0-preview.1 I had to create a separate branch.
change from 2.1.0-preview.1 to 2.1.0 can be seen when [comparing with dev ](https://github.com/microsoftgraph/msgraph-sdk-javascript/compare/dev...nikithauc/Release-2.1.0#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L3). 

I will merge dev with master after this release and push update the version in dev to preview 2.2.0.

Reply:
I see my confusion. I wasn't looking at package.json, and the package.json wasn't matching the tag. This was supposed to be changed in the last release but wasn't.